habitat/config/default.py

```diff
@@ -131,6 +131,36 @@
 _C.TASK.COLLISIONS = CN()
 _C.TASK.COLLISIONS.TYPE = "Collisions"
 # -----------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
+# # EQA TASK
+# -----------------------------------------------------------------------------
+_C.TASK.ACTIONS.ANSWER = CN()
+_C.TASK.ACTIONS.ANSWER.TYPE = "AnswerAction"
+# # EQA TASK QUESTION SENSOR
+# -----------------------------------------------------------------------------
+_C.TASK.QUESTION_SENSOR = CN()
```

Comment:
Should we divide sensors and measures under two different nodes?

Reply:
Make senes will worth to have separate PR for that refactoring.